src/actions/storeActions.ts

```diff
@@ -0,0 +1,45 @@
+import {
+    COLLECTION_ERROR,
+    FETCH_COLLECTIONS,
+    ThunkActionCreator,
+} from './types';
+
+import { logoutUser } from './authActions';
+
+import Config from '../config';
+import Storage from '../storage';
+import { notify } from '../utils';
+
+export const fetchCollections: ThunkActionCreator = () => async (dispatch) => {
+try {
+    const collectionsRes = await fetch(Config.API_URL + Config.routes.store.collection, {
+        method: 'GET',
+        headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+            Authorization: `Bearer ${Storage.get('token')}`,
+        },  
+    });
+
+    const { status } = collectionsRes;
+    if (status === 401 || status === 403) {
+        dispatch(logoutUser());
```

Comment:
hmm why dispatch a logout here versus a notify?

Reply:
If they're not logged in, should we still show the store page and notify them that they're not logged in?

Reply:
Oh interesting, do we not have a protected route from acmucsd.com/{store url} that already blocks people from viewing though?

Reply:
It is protected. As of now, it requires standard access, so no logged out user will see it anyway.

```jsx
            <Route
              exact
              path="/store"
              component={requireStandardAccess(requireAuth(StorePage)) as React.FC}
            />
```

See [index.tsx for routes](https://github.com/annsudhart/membership-portal-ui/blob/87b1b3e2cc6290e487f616a486a59f2bae544c4b/src/index.tsx#L87-L91).

Reply:
I'll remove the code for the dispatch then if it's already taken care of